flask_server.py

```diff
@@ -61,7 +61,7 @@ def get_system_logs():
     log_file_path = settings.get('system_log_file_path')
     if not os.path.exists(log_file_path): return []
     with open(log_file_path, 'r+') as log_file:
-        return [log for log in log_file.readlines()]
+        return list(reversed([log for log in log_file.readlines()]))
```

Comment:
[...].reverse() ?

Reply:
I think it's not working with list comprehension